apiary.apib

```diff
@@ -428,13 +428,14 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### October 2023
 - We added `late_fees` to `invoices.info` and `invoices.list`.
+- We added the `invoices.updateBooked` endpoint.
 
 ### September 2023
 - We added the `meetings.list` and `meetings.info` endpoints.
 - We added `time_estimated` and `amount_unbilled` to `projects-v2/projects.list`.
 - We added sorting on `amount_billed`, `amount_paid`, `amount_unbilled`, `external_budget_spent`, `external_budget`, `internal_budget` and `time_estimated` to `projects-v2/projects.list`.
 
-### August 2023
+#### August 2023
```

Comment:
Q: why do we make this a H4 while the other months are a H3?

Reply:
> Q: why do we make this a H4 while the other months are a H3?

was resolved on master, this brought it back with a merge with master. thanks for noticing

Reply:
The other months are actually H4, it's only the 2 most recent ones that are H3. 🤷 fixing it.